packages/ui/src/app/components/SideBar.tsx

```diff
@@ -99,6 +104,17 @@ export const SideBar = () => {
                 Members
               </NavigationLink>
             </NavigationLinksItem>
+            <NavigationLinksItem>
+              <NavigationLink
+                icon={<PolkadotSymbol />}
+                onClick={() =>
+                  window.open(`https://polkadot.js.org/apps/?rpc=${endpoints.nodeRpcEndpoint}#/explorer`, '_blank')
+                }
+              >
+                Explorer
+                <LinkSymbol color={Colors.Black[500]} />
+              </NavigationLink>
```

Comment:
It would be nicer to have use a hyperlink which shares the `NavigationLink` styles so users don't need to allow a popup